src/core_plugins/kibana/public/management/sections/indices/add_data_steps/parse_csv_step/lib/validate_headers.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+export default function validateHeaders(fields) {
+  if (!_.isArray(fields)) {
+    throw new Error('first argument must be an array');
+  }
```

Comment:
I disagree with that, I still think it's a good idea to enforce your preconditions. Especially when using lodash where many of the functions can operate on both objects and arrays, *not* checking your arguments could lead to hard to find bugs that don't throw errors.

That said, it's not in our styleguide and it's a minor issue here, so it's not worth holding up this PR.